hosted_control_planes/hcp_high_availability/hcp-disaster-recovery-oadp-auto.adoc

```diff
@@ -0,0 +1,82 @@
+include::_attributes/common-attributes.adoc[]
+:context: hcp-disaster-recovery-oadp-auto
+
+toc::[]
+
+On bare metal and {aws-first}, you can automate some backup and restore steps by using the {oadp-first} Operator.
+
+The disaster recovery process with {oadp-first} involves the following steps:
+
+. Preparing your platform, such as {aws-full} or bare metal, to use {oadp-short}
+. Backing up the data plane workload
+. Backing up the control plane workload
+. Restoring a hosted cluster by using {oadp-short}
+
+[id="hcp-auto-dr-prereqs_{context}"]
+== Prerequisites
+
+You must meet the following prerequisites on the management cluster:
+
+* You xref:../../backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc#oadp-installing-operator[installed the {oadp-short} Operator].
+* You created a storage class.
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {oadp-short} subscription through a catalog source.
+* You have access to a cloud storage provider that is compatible with {oadp-short}, such as S3, {azure-full}, {gcp-full}, or MinIO.
+* In a disconnected environment, you have access to a self-hosted storage provider, for example link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/[{odf-full}] or link:https://min.io/[MinIO], that is compatible with {oadp-short}.
+* Your {hcp} pods are up and running.
+
+[id="prepare-aws-oadp-auto_{context}"]
+== Preparing {aws-short} to use {oadp-short}
+
+To perform disaster recovery for a hosted cluster, you can use {oadp-first} on {aws-first} S3 compatible storage.
+
+To prepare {aws-short} to use {oadp-short}, see "Configuring the {oadp-full} with Multicloud Object Gateway".
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc#installing-oadp-aws[Configuring the {oadp-full} with Multicloud Object Gateway]
+
+.Next steps
+
+* Backing up the data plane workload
+* Backing up the control plane workload
+
+[id="prepare-bm-dr-oadp-auto_{context}"]
+== Preparing bare metal to use {oadp-short}
+
+To perform disaster recovery for a hosted cluster, you can use {oadp-first} on bare metal.
+
+To prepare bare metal to use {oadp-short}, see "Configuring the {oadp-full} with AWS S3 compatible storage".
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc#installing-oadp-mcg[Configuring the {oadp-full} with AWS S3 compatible storage]
+
+include::modules/hcp-dr-oadp-dpa.adoc[leveloffset=+1]
+
+[id="backing-up-data-plane-oadp-auto_{context}"]
+== Backing up the data plane workload
+
+If the data plane workload is not important, you can skip this procedure. To back up the data plane workload by using the {oadp-short} Operator, see "Backing up applications".
```

Comment:
I might switch these sentences to change the emphasis: Here is what you can do; and, if you don't want to, then don't. 
```suggestion
To back up the data plane workload by using the {oadp-short} Operator, see "Backing up applications". If the data plane workload is not important, you can skip this procedure. 
```